normalize_all.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# For loop to normalize files from each data source for each language.
+
+for language in af mg
+do
+
+    for data_source in ud um lcc
```

Comment:
Does this not include all of the data sources?

Reply:
Good catch. It looks like I only added a few data sources (and only two langauges) to make sure it worked and then forgot to go back and add them all.

Reply:
Oh haha, nice!  Thanks!